### ISSUE TITLE
Please include Cargo.lock

### ISSUE THREAD

**Issue by wezm** [matheuslessarodrigues/verco#19]
Attempting to build from the 5.0.0 release tarball with `cargo build --release --locked` results in an error:

```
error: the lock file /home/wmoore/Projects/aur/verco/src/verco-5.0.0/Cargo.lock needs to be updated but --locked was passed to prevent this
If you want to try to generate the lock file without accessing the network, use the --offline flag.
```

I use `--locked` when building an [AUR](https://aur.archlinux.org/) package to ensure that the build is reproducible. It appears that you have not committed the `Cargo.lock` file. [This is recommended](https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries) for binary packages to ensure others building from source use the same version of dependencies that you used.

**Comment by vamolessa** [comment]
just merged #20
so will close now!

**Comment by wezm** [comment]
Awesome! Also thanks for the PR @Alphare.
